articles/connections/database/custom-db/templates/create.md

```diff
@@ -439,83 +414,66 @@ function create(user, callback) {
 
   connection.connect();
 
-  var query = "INSERT INTO users SET ?";
+  const query = 'INSERT INTO users SET ?';
 
-  bcrypt.hash(user.password, 10, function (err, hash) {
-    if (err) { return callback(err); }
-    var insert = {
+  bcrypt.hash(user.password, 10, function(err, hash) {
+    if (err) return callback(err);
+
+    const insert = {
       password: hash,
       email: user.email
     };
-    connection.query(query, insert, function (err, results) {
+
+    connection.query(query, insert, function(err, results) {
       if (err) return callback(err);
       if (results.length === 0) return callback();
       callback(null);
     });
   });
-
 }
 ```
 
 ### Oracle
 
 ```sql
 function create(user, callback) {
-
-  var oracledb = require('oracledb');
+  const bcrypt = require('bcrypt');
+  const oracledb = require('oracledb');
   oracledb.outFormat = oracledb.OBJECT;
 
   oracledb.getConnection({
       user: configuration.dbUser,
       password: configuration.dbUserPassword,
-      connectString: "CONNECTION_STRING" // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
+      connectString: 'CONNECTION_STRING' // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
     },
     function(err, connection) {
-      if (err) {
-        return callback(new Error(err));
-      }
-      connection.execute(
-        "SELECT ID, EMAIL, PASSWORD, EMAIL_VERIFIED, NICKNAME" +
-        " FROM Users " +
-        " WHERE EMAIL = :email", [user.email],
-        function(err, result) {
-          if (err) {
-            doRelease(connection);
-            return callback(new Error(err));
-          }
-          if (result.rows.length > 0) {
+      if (err) return callback(err);
+
+      const selectQuery = 'select ID, EMAIL, PASSWORD, EMAIL_VERIFIED, NICKNAME from Users where EMAIL = :email';
```

Comment:
In Heroku review app, code in code block is rendered strangely from here on (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/create#oracle